packages/api/services/deployService.js

```diff
@@ -66,6 +66,15 @@ async function deploy({ name, spaArchive, appPath, ref } = {}) {
     await fileService.write(yamlFilePath, { ref });
   }
 
+  try {
+    const htaccessPath = path.join(tmpDir, ".htaccess");
+    if (!fs.existsSync(htaccessPath)) {
+      await common.htaccess.write(tmpDir, spaConfig);
+    }
+  } catch (error) {
+    console.log("SPA included .htaccess file");
```

Comment:
Hey @kunyan This error message seems to be a bit too generic and confusing for me. Can we make it a bit more explicit?

Reply:
any suggestion ?

Reply:
Something like what @Jared-Sprague has suggested in his comment.